src/app/core/services/repository/record-repository.service.spec.ts

```diff
@@ -9,3 +16,148 @@ describe('RecordRepositoryService', () => {
     expect(service).toBeTruthy();
   });
 });
+
+fdescribe('save()', () => {
+  var originalTimeOut;
+
+  beforeEach(function() {
+    originalTimeOut = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 1000000;
+  })
+  
+  beforeEach(() => TestBed.configureTestingModule({}));
+
+  afterEach(async () => await Storage.clear());
+
+
+  it('should save record', async (done) => {
+    const service: RecordRepositoryService = TestBed.get(RecordRepositoryService);
+
+    const record = new Record(20191208);
+
+    const recordOutput = JSON.stringify(record);
+    
+    service.save(record).subscribe({ next(x) {
```

Comment:
You don't have to write `next(x)`

It will be cleaner this way:

```
service.save(record).subscribe(x => {
      expect(JSON.stringify(x[x.length - 1])).toEqual(recordOutput);
      done();
});
```